### PR TITLE
Moved JSX content from dashboard to map and made minor changes to JSX content to reflect backend keywords

### LINKED ISSUE #3: Dashboard Page
Start with mobile, then tablet, and finish with desktop

### LINKED ISSUE #91: Merge dashboard and map pages

### PR DESCRIPTION
**Pull request by trca831**
Closes #91

### CODE REVIEW

**Review by Count-MonteCristo — APPROVED** [review]
Just need to delete the unused page.

File: src/components/pages/Map.js [hunk 1]
```diff
@@ -142,7 +153,75 @@ const MapComponent = () => {
...
   return (
     <div className="map-container">
-      <div className="feed-container">{/* Add code for the feed here */}</div>
+      <div className="feed-container">{
+        <>
+        <button
+          type="button"
+          className="edit-filter-button"
+        >
+          {" "}
```
Comment (dkprioleau) [inline src/components/pages/Map.js:162]: remove

Comment (dkprioleau) [reply src/components/pages/Map.js:162]: You can remove all the empty braces

**Review by dkprioleau — APPROVED** [review]
I think it would be a good idea to figure out what you're going to show on the dashboard side when the user has not submitted a zipcode yet so it's not just white space

**Pull request merged**